packages/drivers/redux-little-router/misc/test-suite.js

```diff
@@ -0,0 +1,244 @@
+/* eslint-env jest */
+import { compose, applyMiddleware } from 'redux'
+import { routerForBrowser, push } from 'redux-little-router'
+import { createStore as krmlCreateStore, when } from 'k-ramel'
+import omit from 'lodash/omit'
+
+export default (driver) => {
+  const createStore = (description, options, routes) => {
+    const { reducer, middleware, enhancer } = routerForBrowser({ routes })
```

Comment:
You should wrap this in the driver.

 - `driver.getEnhancer`
 - `driver.getReducer`